modules/update-using-custom-machine-config-pools-mcp-remove.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * updating/update-using-custom-machine-config-pools.adoc
+
+[id="update-using-custom-machine-config-pools-mcp-remove_{context}"]
+= Moving a node to the original machine config pool
+
+In this canary rollout update process, after you have unpaused a custom machine config pool (MCP) and verified that the applications on the nodes associated with that MCP are working as expected, you should move the node back to its original MCP by removing the custom label you added to the node.
+
+[IMPORTANT]
+====
+A node must have a role to be properly functioning in the cluster.
+====
+
+To move a node to its original MCP
+
+. Ensure that the nodes have a `worker` label or a label from an MCP that is updated.
++
+[source,terminal]
+----
+$ oc label node ci-ln-0qv1yp2-f76d1-kl2tq-worker-a-j2ssz node-role.kubernetes.io/worker=
+----
++
+.Example output if the `worker` label is present:
++
+[source,terminal]
+----
+error: 'node-role.kubernetes.io/worker' already has a value (), and --overwrite is false
+----
++
+If the node does not have a `worker` label or a label from an updated MCP, add the label. 
+
+. Remove the custom label from the node. 
++
+[source,terminal]
+----
+$ oc label node <node_name> node-role.kubernetes.io/<custom-label>-
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc label node ci-ln-0qv1yp2-f76d1-kl2tq-worker-a-j2ssz node-role.kubernetes.io/workerpool-canary-
+----
++
+.Example output
++
+----
+node/ci-ln-0qv1yp2-f76d1-kl2tq-worker-a-j2ssz labeled
+----
++
+The MCO moves the node(s) back to the original MCP and reconciles the node to the MCP configuration:
++
+[source,terminal]
+----
+$oc get mcp
+----
++
+[source,terminal]
+----
+NAME                CONFIG                                                   UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+master              rendered-master-1203f157d053fd987c7cbd91e3fbc0ed         True      False      False      3              3                   3                     0                      61m
+workerpool-canary   rendered-mcp-noupdate-5ad4791166c468f3a35cd16e734c9028   True      False      False      0              0                   0                     0                      21m
+worker              rendered-worker-5ad4791166c468f3a35cd16e734c9028         True      False      False      3              3                   3                     0                      61m
```

Comment:
We'd better to explain that it will take some time to update the machine count of the custom MCP. During my test, i also hit the issue in https://github.com/openshift/openshift-docs/pull/34445#discussion_r681027050. It tooks me about 20mins, during the time, the output of `oc get mcp` looks not correct that the total machine counts is not the expected as following:
```
# ./oc get mcp
NAME                CONFIG                                                        UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
master              rendered-master-24d50281efc91dd219043c37bb86ffce              True      False      False      3              3                   3                     0                      8h
worker              rendered-worker-8ee5ad95bfdb0175a7aaa5566f778d6f              True      False      False      3              3                   3                     0                      8h
workerpool-canary   rendered-workerpool-canary-8ee5ad95bfdb0175a7aaa5566f778d6f   True      False      False      1              1                   1                     0                      5h33m

```
From above output, it looks like there are 4 worker machines.